mlir/include/mlir/Analysis/PresburgerSet.h

```diff
@@ -90,6 +90,11 @@ class PresburgerSet {
   /// any of the FACs in the union are unbounded.
   bool findIntegerSample(SmallVectorImpl<int64_t> &sample);
 
+  /// eliminates FlatAffineConstraints, that are fully contained within other
```

Comment:
I think this could be written better. e.g.

```quote
Simplifies the representation of the PresburgerSet.

In particular, removes all FACs which are subsets of other FACs in the set.
```